paper.md

```diff
@@ -60,7 +60,7 @@ such as data races and fatal use-after-free errors.
 
 Rust also allows compatibility with the C ABI, allowing for simple interoperability
 with C, and most other languages by extension. Once a design has been properly fleshed out,
-we plan to expose a C API, along with bindings to other languages (Currently planned are C, C++, Python and Julia bindings).
+we plan to expose a C API, along with bindings to other languages (C, C++, Python and Julia bindings are currently planned).
```

Comment:
My understanding is that there are some R and Python packages that are already using Faer, though I don't think there are comprehensive bindings.

Reply:
yeah, i think there's people using it but i haven't really been following their progress